chemprop/uncertainty/predictor.py

```diff
@@ -19,8 +23,27 @@ def _calc_prediction_uncertainty(
     ) -> tuple[Tensor, Tensor]:
         """
         Calculate the uncalibrated predictions and uncertainties for the dataloader.
+
+        dataloader: DataLoader
+            the dataloader used for model predictions and uncertainty predictions
+        models: Iterable[MPNN]
+            the models used for model predictions and uncertainty predictions
+        trainer: pl.Trainer
+            an instance of the :class:`~lightning.pytorch.trainer.trainer.Trainer` used to manage model inference
+
+        Returns
+        -------
+        preds : Tensor
+            the model predictions, with shape varying by task type:
+
+            - ``(n, t, m)`` for regression or binary classification,
+            - ``(n, t, c, m)`` for multiclass classification,
```

Comment:
I don't care what style you use for shape annotations (though I do prefer ``n x m x ...``), but I just ask that you be consistent.